### DISCUSSION TITLE
Existing UAS published standards or examples?

### DISCUSSION THREAD

**Discussion by serbinsh** [ess-dive-community/essdive-uas#2] [General]
We (Kim) have identified some existing institutional UAS data / metadata standards, including:

ARM File Data Standards: https://www.arm.gov/publications/programdocs/doe-sc-arm-15-004.pdf

EPA UAS Handbook: https://www.epa.gov/system/files/documents/2021-10/epa-uas-handbook-may-2021-public-website-1.pdf

USGS Data Delivery Specifications: https://d9-wret.s3.us-west-2.amazonaws.com/assets/palladium/production/s3fs-public/atoms/files/USGS%20UAS%20Data%20Delivery%20Spec1.4_042021.pdf

NASA Procedural Requirements: https://nodis3.gsfc.nasa.gov/npg_img/N_PR_7900_003D_/N_PR_7900_003D__Chapter5.pdf

But are there others we can review?  What about other non-Agency examples?

**Comment by charuleka** [comment]
Have you looked into OGC and FGDC spatial data standards? See for e.g.
https://share.ansi.org/Shared%20Documents/Standards%20Activities/UASSC/May%2019,%202017%20ANSI%20UAS%20Standardization%20Coordination%20Meeting/Percivall_OGC_ANSI_UAS_May_2017.pdf

**Reply by serbinsh** [reply]
Hey @charuleka thanks for the feedback - however I should clarify, I am speaking about metadata/reporting formats for UAS and not suggesting that we change or redo existing FGDC or spatial data standards.  Instead we will be using these standards with our reporting format to guide how units, projects, etc are/should be expressed.  

What I am talking about here is guidance, or what we are finding, lack there of from other agencies right now

**Reply by serbinsh** [reply]
As even the ESIP front page suggests "As a new technology, however, there are currently no industry-wide accepted best practices for sUAS sensor and flight data handling and management. "

**Reply by serbinsh** [reply]
And we need to separate file format standards from reporting format standards, though we anticipate providing file format guidance (e.g. COG, geojson, txt depending) it is not the main thrust of this effort

**Comment by charuleka** [comment]
There is also an research data alliance interest group on UAS data:
https://www.rd-alliance.org/groups/small-unmanned-aircraft-systems%E2%80%99-data-ig
, and an ESIP (earth systems information partners) cluster for drone data -
https://wiki.esipfed.org/Drone_Cluster

**Comment by charuleka** [comment]
I understand. It would still make sense to review or at least be aware of
other parallel UAS efforts to make sure our metadata reporting formats will
be compatible with the OGC requirements and RDA recommendations. Our goal
is to propose the minimum set of metadata that is needed in our reporting
formats, so if the OGC/RDA data standards require some metadata, we want to
be additive not duplicative.

**Comment by serbinsh** [comment]
Here is a another good resource:

https://www.biorxiv.org/content/10.1101/2021.02.05.430004v1

https://www.sciencedirect.com/science/article/abs/pii/S1550742418302914?via%3Dihub

How these authors organized their 
https://datacommons.cyverse.org/browse/iplant/home/shared/commons_repo/curated/Gillan_et_al_RAMA_2019

And their data abstract: https://de.cyverse.org/anon-files//iplant/home/shared/commons_repo/curated/Gillan_et_al_RAMA_2019/Gillan_etal_2019__RAMA_abstract.pdf

**Comment by serbinsh** [comment]
We can find another example in the HiLDEN protocols

https://arcticdrones.org/protocols/

Specifically: https://droneecology.files.wordpress.com/2019/06/hilden-metadata-2019.pdf

**Comment by dlebauer** [comment]
Two papers by @r4space et al  
- Wyngaard, J.; Barbieri, L.; Thomer, A.; Adams, J.; Sullivan, D.; Crosby, C.; Parr, C.; Klump, J.; Raj Shrestha, S.; Bell, T. Emergent Challenges for Science sUAS Data Management: Fairness through Community Engagement and Best Practices Development. Remote Sens. 2019, 11, 1797. https://doi.org/10.3390/rs11151797
- Thomer, Andrea, et al. "A Minimum Information Framework for capturing FAIR data with small Uncrewed Aircraft Systems." (2021).  https://doi.org/10.31223/X5Z338

**Comment by serbinsh** [comment]
And of course the TERRA-REF project put a lot of work into data formatting and standardization that is relevant to this effort: https://github.com/terraref

I expect there are things we can build on from that effort and that perhaps can feed back to TERRA-REF

**Comment by bgreene777** [comment]
One fairly recent example is a collection of datasets from the LAPSE-RATE field campaign published in a special issue of Earth System Science Data (ESSD): https://essd.copernicus.org/articles/special_issue1014.html 

The data formats are described in this summary paper:
de Boer, G., and Coauthors, 2020: Data generated during the 2018 LAPSE-RATE campaign: an introduction and overview. Earth Syst. Sci. Data, 12, 3357–3366, https://doi.org/10.5194/essd-12-3357-2020.
